src/SpectatorView.Unity/Assets/SpectatorView/Scripts/Compositor/CompositorWorldAnchorLocalizationManager.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView.WorldAnchors;
+using System;
+using System.Collections;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView.Compositor
+{
+    /// <summary>
+    /// Component responsible for requesting localization of shared coordinates on devices
+    /// connected to the compositor. Shared coordinates are persisted using
+    /// the <see cref="WorldAnchorSpatialLocalizer"/> after they are resolved, and
+    /// are restored upon initial connection.
+    /// </summary>
+    public class CompositorWorldAnchorLocalizationManager : Singleton<CompositorWorldAnchorLocalizationManager>
```

Comment:
>CompositorWorldAnchorLocalizationManager [](start = 17, length = 40)

Do we want to scope this class to only exist in editor?

Reply:
I could move this to be in the Editor directory so that it's implicitly editor-only

---
In reply to: [295548709](https://github.com/microsoft/MixedReality-SpectatorView/pull/26#discussion_r295548709) [](ancestors = 295548709)

Reply:
On second thought I think I'll hold off on that. The rest of the compositor-related scripts are not editor-only currently and I think we should think about that more-holistically.

---
In reply to: [295555617](https://github.com/microsoft/MixedReality-SpectatorView/pull/26#discussion_r295555617) [](ancestors = 295555617,295548709)